### DISCUSSION TITLE
Can´t install on new nspanel

### DISCUSSION THREAD

**Discussion by famviuf** [Blackymas/NSPanel_HA_Blueprint#2047] [Q&A]
Hello
Have 4 panels with
TFT Version
4.3.4
ESPHome Version
4.3.4
Blueprint Version
4.3.4
Panel Model
EU
Works fine.
Now i will install a new nspanel using a serial cable.
My nspanel_esphome.yaml is:

![Picture](https://github.com/Blackymas/NSPanel_HA_Blueprint/assets/65901152/af63aedb-c67d-4685-a1ab-63c3bbaf6ae6)

Flashing is going good, no problem
But when i power it on, the screen flash "sonoff" 3 times and no more.
??
What do i do wrong?

**Comment by edwardtfn** [comment]
- #2052
